doc/lightning_tensor/device.rst

```diff
@@ -19,12 +19,12 @@ The default setup for the MPS tensor network approximation is:
 The ``lightning.tensor`` device dispatches all operations to be performed on a CUDA-capable GPU of generation SM 7.0 (Volta)
 and greater.
 
-.. note:: 
-    Given the inherent parallelism of GPUs, simulations with intensive parallel computation, such as those with larger maximum
-    bond dimensions, stand to gain the most from the computational power offered by GPU and those simulations can benifit from the 
-    ``lightning.tensor`` device.  It's worth noting that if the bond dimension used in the simulation is small, the ``lightning.tensor`` 
-    device with ``MPS`` running a GPU may perform slower compared to a ``default.tensor`` device with ``MPS`` running on a CPU. For more details
-    on how bond dimension affects the simulation performance, please refer to the ``Approximate Tensor Network Methods`` section in the `cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`__.
+Some tips on the usage of the ``lightning.tensor`` device:
```

Comment:
Please integrate these points here
> This device supports both analytical and finite shots measurements. Currently, the supported differentiation methods are ``parameter-shift`` and ``finite-diff``. Note that MPS backend of ``lightning.tensor`` supports multi-wires gate via Matrix Product Operators (MPO). Disable the new arithmetic operator dunders is recommended since only 1-wire Hermitian is supported by ``lightning.tensor``. It is recommended to use the shot measurements for the ``lightning.tensor`` device. The calculation of analytical probabilities and variances may result in excessive memory usage for large-scale quantum simulations or become impractical due to prohibitive computational costs. It's also recommended to disable ``new_opmath`` for the ``lightning.tensor`` device since ``lightning.tensor`` only supports 1-wire Hermitian observables.